src/main/kotlin/com/openlattice/chronicle/storage/ChroniclePostgresTables.kt

```diff
@@ -97,15 +97,13 @@ class ChroniclePostgresTables {
         val PARTICIPANTS = PostgresTableDefinition("participants")
             .addColumns(
                 PARTICIPANT_ID,
-                TITLE,
-                NAME,
+                FRIENDLY_ID,
                 FIRST_NAME,
                 LAST_NAME,
-                DATE_OF_BIRTH,
-                DESCRIPTION,
-                SETTINGS
+                NAME,
+                DATE_OF_BIRTH
             )
-            .primaryKey(PARTICIPANT_ID)
+            .primaryKey(PARTICIPANT_ID, FRIENDLY_ID)
 
```

Comment:
I think we need to enforce unique friendly_id constraint at study level since this is the value that is displayed on the participant's table in the web app. (Friendly_id, study_id) seems more suitable as the primary key

Reply:
Yes. That's what I intended, I think I just put this in the wrong place.